File: CodinGame.Application/client/src/views/mars-lander/api/requests/LandRequest.ts
import {Map} from "@/views/mars-lander/models/environment/Map";
import {AiWeight} from "@/views/mars-lander/models/evolution/AiWeight";
import {EvolutionParameters} from "@/views/mars-lander/models/evolution/EvolutionParameters";

export class LandRequest {
  public Map!: Map
  public AiWeight!: AiWeight;
  public Parameters!: EvolutionParameters;

  constructor(init?: Partial<LandRequest>) {
    Object.assign(this, init);
  }
}

File: CodinGame.Application/Models/LandRequest.cs
using CodinGame.MarsLander.Models;
using CodinGame.MarsLander.Models.Dtos;

namespace CodinGame.Application.Models
{
    public class LandRequest
    {
        public Map Map { get; set; }
        public AiWeight AiWeight { get; set; }
        public EvolutionParameters Parameters { get; set; }
    }
}